### ISSUE TITLE
V5 setup requires non-SSL urls & a temp proxy conf

### ISSUE THREAD

**Issue by souldessin** [invoiceninja/dockerfiles#295]
First off, thank you to the IN team. I'm looking forward to v5 rocking even more than v4.

---

*tl:dr* - SSL can't be setup & relied upon when setting up v5. All redirects with a reverse proxy must be available on the non-ssl side of the reverse proxy. In addition, the PDF test won't work on SSL at all.

## My Setup

I am running Debian with IN:v5 through Docker-Compose and served next to a bunch of other dockerized services through Apache2.

## Issue

The base app is passing only non-ssl links for the `app.js`, `setup.js`, and `style.css` files in the html. It necessitates a temporary reverse proxy setup on the server in order to finish the setup.

This issue also requires that setup be done solely through the non-ssl address for the PDF generator test to work, as the script makes the call for the /setup/check_pdf address without https url and can't handle the redirect from a server without the temporary non-ssl proxy.

## Recommended Solution

I think there should be a single page form, with inline styling, that asks if HTTPS should be forced. Then it loads the normal setup from there, with the correct URLs being passed.
